docs/notebooks/data_transformation.pct.py

```diff
@@ -244,6 +255,71 @@ def normalise(x, mean=None, std=None):
 print(f"observation: {observations[arg_min_idx, :]}")
 
 
+# %% [markdown]
+# # Data transformation with the DataTransformModelWrapper class
+#
+# Actually, we can achieve the same effect with a standard `BayesianOptimizer` by using a `DataTransformModelWrapper`.
+#
+# First, we create a new model class that inherits from the model wrapper class and the kind of `TrainableProbabilisticModel` that we want to use.
+
+# %%
+from trieste.models.normalization import DataTransformModelWrapper
+
+class GPRwithDataNormalization(DataTransformModelWrapper, GaussianProcessRegression):
```

Comment:
I'm thinking we should offer these so users don't need to define them, say in `models/gpflow/transformers.py`, this is easy to maintain...

Reply:
I agree, there were actually a few methods that aren't part of the common interface which I've now wrapped for the appropriate classes. (The main one is `covariance_between_points` for `GaussianProcessRegression`)